pkg/backend/openshiftcluster/create.go

```diff
@@ -322,7 +327,26 @@ func (m *Manager) Create(ctx context.Context) error {
 		return err
 	}
 
-	return i.Install(ctx, installConfig, platformCreds, image)
+	gcskey, gcscert := m.env.ClustersGenevaLoggingSecret()
```

Comment:
move to separate helper function. `boostrapLoggingCreds, err:= getBoostrapLoggingCreds`